src/elements/core/styles/mixins/image.scss

```diff
@@ -0,0 +1,34 @@
+@use './typo';
+
+@mixin figure {
+  display: grid;
+  grid-template-rows: auto;
+  grid-template-columns: 100%;
+  grid-auto-rows: auto;
+  margin: 0;
+  height: 100%;
+}
+
+@mixin figure-image {
+  grid-row: 1;
+  grid-column: 1;
+  width: 100%;
+}
+
+@mixin figure-caption {
+  grid-row: 2;
+  grid-column: 1;
+  padding-block-start: var(--sbb-spacing-fixed-4x);
+  @include typo.text-xs--regular;
+}
+
+@mixin figure-overlap-base {
+  z-index: 1;
```

Comment:
Nice catch. I want to understand better how stacking contexts work because I believe there should be a way to create a different stacking context for the picture.
Anyway, we go with the order for the time being :)

Reply:
Edit: the order solution does not work in the teaser product :/

Reply:
Did you use order and position: relative in combination? If you like, I can deep dive into this specific issue?

Reply:
Since this is the last 2.0 pending thing I'll gladly accept the help 😅